pom.xml

```diff
@@ -13,6 +16,20 @@
       <distribution>repo</distribution>
     </license>
   </licenses>
+  <developers>
+    <developer>
+      <id>amazonwebservices</id>
+      <organization>Amazon Web Services</organization>
+      <roles>
+        <role>developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:git@github.com:aws/amazon-documentdb-jdbc-driver.git</connection>
+    <developerConnection>scm:git:git@github.com:aws/amazon-documentdb-jdbc-driver.git</developerConnection>
+    <url>git@github.com:aws/amazon-documentdb-jdbc-driver.git</url>
```

Comment:
need to add just link to repo: `https://github.com/aws/amazon-documentdb-jdbc-driver`